Changelog.md

```diff
@@ -46,7 +46,7 @@ English
 - **09-30-2020**
 
     - Support for Big Sur Beta 9 (20A5384c)
-    - Added macOS Boot chime support: disabled by default, set `PlayChime` and `AudioSupport` to `True` if you want it. Boot chime was upsampled by @tylernguyen using Audacity: this upsampled file will be used until `AudioDxe.efi` can upsample audio on the fly.
+    - Added macOS Boot chime support: disabled by default, set `PlayChime` and `AudioSupport` to `True` if you want it. Also add AudioDxe.efi to >Drivers inside config.plist. Boot chime was upsampled by @tylernguyen using Audacity: this upsampled file will be used until `AudioDxe.efi` can upsample audio on the fly.
```

Comment:
I think it is on by default tbh. Using the 1.6.0 release atleast

Reply:
Not on current release.

Config has it omitted by default, however its required to enable the chime sound

As it stands, 

<key>Drivers</key>
		<array>
			<string>HfsPlus.efi</string>
			<string>OpenRuntime.efi</string>
		</array>